virtual_undercloud.yml

```diff
@@ -19,14 +19,31 @@
     - name: clean previous vm
       shell: |
          source {{ infrared_dir }}/.venv/bin/activate
-         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url.rhel7 }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
```

Comment:
add a task for setting the image url and pass that to the previous tasks, no need to add a clean and create vm for each rhel image. it will be more clean and easy to maintain